app/client/components/commandList.ts

```diff
@@ -470,6 +471,10 @@ export const groups: CommendGroupDef[] = [{
       keys: ['Mod+C'],
       desc: 'Copy current selection to clipboard',
       bindKeys: false,
+    }, {
+      name: 'contextMenuCopyWithHeaders',
+      keys: ['Mod+Shift+C'],
```

Comment:
Since we don't have good idea here, can you just remove this shortcut? It is ok not to have it.
We can add it later after doing some more research on it.

Reply:
Done.